definitions/private-law/json/AuthorisationCaseState.json

```diff
@@ -343,5 +343,41 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "PRLAPPS",
+    "CaseStateID": "PROCEEDS_IN_HERITAGE_SYSTEM",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-solicitor",
+          "citizen",
```

Comment:
Only CRU for systemupdate, rest of the world only R

Reply:
Changed as per provided comment